### PR TITLE
taxonomy: remove duplicate (and wrong translation)

### PR DESCRIPTION
**Pull request by OpenChris**
fr:Steaks de bœuf if not Beef patties

### CODE REVIEW

**Comment by stephanegigandet** [comment]
> fr:Steaks de bœuf if not Beef patties

well just above we have:

<en:Meats
en:Meat patties, Steaks
de:Steaks
es:Filetes
fi:pihvit
fr:Steaks de viande, steaks, steacks
he:סטייקים
it:Bistecche
nl:Steaks
zh:肉排

And there's another entry:

<en:Beef
<en:Steaks
en:Beef steaks
de:Rindersteaks
es:Filetes de ternera
fi:naudanpihvit
fr:Steaks de bœuf, steak de bœuf
it:Bistecche di manzo
nl:Biefstukken
pl:Stek wołowy
zh:牛排

So if "patties" = "steaks", then the en:beef-patties should just be a synonym for beef steaks?

I think there's a larger issue with the entry <en:Meats
en:Meat patties, Steaks.
Patties = Ground steaks, if it's not ground, it's not a patty.

**Comment by OpenChris** [comment]
Agree with @stephanegigandet : not ground => not patty.
The categ hierarchy needs a larger review.
You may want to delay this PR until then - however it does solve a duplicate issue on fr:Steaks de bœuf

**Comment by OpenChris** [comment]
@stephanegigandet @moon-rabbitOFF @aleene I have cleaned the "Meat patties" to separate a "Steaks" hierarchy with regular beef steaks and all, and a "Patties" hierarchy with Meat patties, Veg patties, Beef patties (no children), Fish, chicken etc...

**Review by stephanegigandet — COMMENTED** [review]
Looks good to me.

I agree with @moon-rabbitOFF (from Slack discussion) : "Galettes au poulet" might be more accurate than "Galettes de poulet".

**Comment by moon-rabbitOFF** [comment]
Great! It is the same for "galettes au poisson" and "galettes au boeuf"

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -70073,7 +70073,15 @@ it:Bistecche
 nl:Steaks
 zh:肉排
 
+<en:Meats
+en:Meat patties
+fr:Galettes de viande
+
 <en:Meat patties
+en:Beef patties
+fr:Galettes de bœuf
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:70082]:
```suggestion
fr:Galettes au bœuf
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -70073,7 +70073,15 @@ it:Bistecche
 nl:Steaks
 zh:肉排
 
+<en:Meats
+en:Meat patties
+fr:Galettes de viande
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:70078]:
```suggestion
fr:Galettes à la viande
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -81899,6 +81907,7 @@ wikidata:en:Q600396
 # Fish patties are generally called "fish burgers", but it is not put as a synonym since it could be confused with "fish hamburgers"
 <en:Fishes
 en:Fish patties
+fr:Galettes de poisson
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:81910]:
```suggestion
fr:Galettes au poisson
```

Comment (OpenChris) [reply taxonomies/categories.txt:81910]: @stephanegigandet @moon-rabbitOFF you'll have a duplicate with :
><en:Meals
en:Crepes filled with fish, Buckwheat crepes filled with fish
fr:Crêpes au poisson, Galettes au poisson, Crêpes fourrées au poisson, Galettes fourrées au poisson
agribalyse_food_code:en:25625
ciqual_food_code:en:25625
ciqual_food_name:en:Crepe or buckwheat crepe, filled with fish and/or seafood
ciqual_food_name:fr:Crêpe ou Galette fourrée au poisson et / ou fruits de mer

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**